backend/aws-parameterstore.go

```diff
@@ -5,27 +5,28 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
 )
 
 // AWSParameterStore implements Backend for AWS Parameter Store service.
 type AWSParameterStore struct {
-	client   *ssm.SSM
+	client   *ssm.Client
 	kmsKeyID string
 }
 
 // NewAWSParameterStore returns a new AWSParameterStore with an initialized AWS SSM client.
 func NewAWSParameterStore(ctx context.Context, p *Profile) (*AWSParameterStore, error) {
-	sess, err := session.NewSession(&aws.Config{
-		Region:     aws.String(p.Location),
-		MaxRetries: aws.Int(2),
-	})
+	// Load the Shared AWS Configuration (~/.aws/config)
+	cfg, err := config.LoadDefaultConfig(context.TODO())
```

Comment:
There is a context passed as an input parameter, why do a `context.TODO()` instead of passing the `ctx`?